src/main/kotlin/com/dumbdogdiner/stickysurvival/listener/GameTriggersListener.kt

```diff
@@ -0,0 +1,160 @@
+package com.dumbdogdiner.stickysurvival.listener
+
+import com.dumbdogdiner.stickysurvival.StickySurvival
+import com.dumbdogdiner.stickysurvival.event.TributeAddEvent
+import com.dumbdogdiner.stickysurvival.event.TributeRemoveEvent
+import com.dumbdogdiner.stickysurvival.util.messages
+import com.dumbdogdiner.stickysurvival.util.safeFormat
+import com.sk89q.worldedit.LocalSession
+import com.sk89q.worldedit.WorldEdit
+import com.sk89q.worldedit.bukkit.BukkitAdapter
+import com.sk89q.worldedit.command.tool.SelectionWand
+import com.sk89q.worldedit.world.item.ItemType
+import com.thevoxelbox.voxelsniper.VoxelProfileManager
+import org.bukkit.Bukkit
+import org.bukkit.Material
+import org.bukkit.entity.Player
+import org.bukkit.event.EventHandler
+import org.bukkit.event.Listener
+import org.bukkit.inventory.ItemStack
+
+/**
+ * GameTriggersListener
+ *
+ * Internal event listener for hooking into other plugins
+ * (eg. to disable external plugin functionality during games)
+ */
+object GameTriggersListener : Listener {
+
+    // Simple class for easier debug logging
+    private class GTLLogger(val topic: String) {
+        fun log(msg: String) {
+            StickySurvival.instance.logger.info("[GameTriggers $topic]: $msg")
+        }
+    }
+
+    //region Utils (VoxelSniper, WorldEdit)
+
+    private fun getVoxelProfileManager(): VoxelProfileManager? {
+        // Create a instance of GTLLogger for logging
+        val logger = GTLLogger("getVoxelProfileManager")
+
+        if (Bukkit.getServer().pluginManager.isPluginEnabled("VoxelSniper")) {
+            logger.log("plugin enabled!")
+            // Make sure the VoxelSniper plugin is loaded before trying to get VoxelProfileManager
+            val voxelProfileManager = VoxelProfileManager.getInstance()
+
+            logger.log("is null: ${(voxelProfileManager == null)}")
+
+            if (voxelProfileManager != null) {
+                logger.log("found voxelProfileManager!")
+                // VoxelProfileManager is available, return the instance of it
+                return voxelProfileManager
+            } else {
+
+                logger.log("plugin enabled but instance is null!")
+                // For whatever reason, the plugin is enabled but the instance is null.
+                // This shouldn't happen!
+                return null
+            }
+        } else {
+
+            logger.log("plugin is not loaded!")
+            // Plugin is not loaded
+            return null
+        }
+    }
+
+    private fun getWorldEditLocalSession(player: Player): LocalSession? {
+        // Make sure WorldEdit is actually enabled
+        if (Bukkit.getServer().pluginManager.isPluginEnabled("WorldEdit")) {
+            val actor = BukkitAdapter.adapt(player)
+            val sessionManager = WorldEdit.getInstance().sessionManager
+            return sessionManager.get(actor)
```

Comment:
You can use `[]` instead of `.get()` here if you'd like (Kotlin automatically adds operator overloads for Java libraries with suitable method names)